libraries/chain/webassembly/eos-vm-oc/intrinsic.cpp

```diff
@@ -0,0 +1,19 @@
+#include <eosio/chain/webassembly/eos-vm-oc/intrinsic.hpp>
+
+namespace eosio { namespace chain { namespace eosvmoc {
+
+static intrinsic_map_t& the_intrinsic_map() {
+   static intrinsic_map_t intrinsic_map;
+   return intrinsic_map;
+}
+
+const intrinsic_map_t& get_intrinsic_map() {
+   return the_intrinsic_map();
+}
+
+intrinsic::intrinsic(const char* n, const IR::FunctionType* t, void* f, size_t o) {
+   the_intrinsic_map().erase(n);
```

Comment:
Is it possible for ``n`` to already exist in the map?

Reply:
It is. First all intrinsics are registered and then the internals of EOS-VM OC come along and overwrite eosio_exit

Reply:
Oh, so that's why you're setting the static initialization order.

Reply:
That could use some explanation.